pom.xml

```diff
@@ -32,7 +32,7 @@
   <name>Apache Accumulo Testing</name>
   <description>Testing tools for Apache Accumulo</description>
   <properties>
-    <accumulo.version>2.1.0-SNAPSHOT</accumulo.version>
```

Comment:
I wonder if changing this to 2.1.1-SNAPSHOT would have also fixed this, maybe those are available on the apache snapshot repo, not sure.

Reply:
It is possible, but this matches what is in the main accumulo repo.